src/media/css/containers/addon-dashboard-detail.styl

```diff
@@ -0,0 +1,28 @@
+@import '../lib'
+
+
+.addon-dashboard-detail .page--main
+.addon-dashboard-detail h1
+  max-width 800px
```

Comment:
Totally random and coincidental but I should probably just var a few different page widths just to standardize a little.